src/utils/firebaseConfig.js

```diff
@@ -0,0 +1,8 @@
+export const firebaseConfig = {
```

Comment:
Config should not be committed, git ignore out of date as config location has been moved.